Fixx/5.3/doc m15 m16 m17 (#196)
exoplatform/exo-documentation: docs/GettingStarted.rst

Sharing a News
---------------

Once an article is posted, you will be allowed to share it in other spaces. The sharing action is possible from the article details page and from News application. 
When you click on the share icon, a drawer appears containing a reminder of the article's title, its creation user and its original space. 

> You have to select the destination space(s) and you can optionaly add a description which will be displayed in the sharing activity.

Reviewer: ```suggestion
You have to select the destination space(s) and you can optionally add a description which will be displayed in the sharing activity.
```

|image167|

> When you display the activity details, you will see the information about the sharing user, destination space and the description if added in the button of the page:

Reviewer: ```suggestion
When you display the activity details, you will see the information about the sharing user, destination space and the description (if any) at the bottom of the page:
```
